### PR TITLE
Many doc updates

### PR DESCRIPTION
**Pull request by justin808**
1. Renamed basics to guides
2. Moved Rails/Webpacker React Integration Options to be inside of
   "guides"

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1367)

### CODE REVIEW

**Review by Judahmeek — COMMENTED** [review]
> Reviewed 30 of 30 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1367)* status: all files reviewed, 1 unresolved discussion (waiting on @justin808)

___
*[README.md, line 54 at r2](https://reviewable.io/reviews/shakacode/react_on_rails/1367#-MhMgX9o1fquitPap_GZ:-MhMgX9o1fquitPap_G_:b-dbcud6) ([raw file](https://github.com/shakacode/react_on_rails/blob/9ef6b326213480d0022db42710e5517cad0317bc/README.md#L54)):*
> ```Markdown
> Feel free to contact Justin Gordon, [justin@shakacode.com](mailto:justin@shakacode.com), maintainer of React on Rails, for more information.
> 
> [Click to join **React + Rails Slack**](https://reactrails.slack.com/join/shared_invite/enQtNjY3NTczMjczNzYxLTlmYjdiZmY3MTVlMzU2YWE0OWM0MzNiZDI0MzdkZGFiZTFkYTFkOGVjODBmOWEyYWQ3MzA2NGE1YWJjNmVlMGE).
> ```

Are we linking to the ReactRails slack somewhere else?

File: README.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ Given that `rails/webpacker` gem already provides basic React integration, why w
 1. Tight integration with [rails/webpacker](https://github.com/rails/webpacker).
 1. Server-Side Rendering (SSR), often used for SEO crawler indexing and UX performance, is not offered by `rails/webpacker`.
 1. [Redux](https://github.com/reactjs/redux) and [React Router](https://github.com/ReactTraining/react-router#readme) integration with server-side-rendering.
-1. [Internationalization (I18n) and (localization)](https://www.shakacode.com/react-on-rails/docs/basics/i18n)
+1. [Internationalization (I18n) and (localization)](https://www.shakacode.com/react-on-rails/docs/guides/i18n)
 1. A supportive community. This [web search shows how live public sites are using React on Rails](https://publicwww.com/websites/%22react-on-rails%22++-undeveloped.com+depth%3Aall/).
 1. [Reason ML Support](https://github.com/shakacode/reason-react-on-rails-example).
 
-See [Rails/Webpacker React Integration Options](docs/guides/rails-webpacker-react-integration-options.md) for comparisons to other gems.
+See [Rails/Webpacker React Integration Options](https://www.shakacode.com/react-on-rails/docs/guides/rails-webpacker-react-integration-options.md) for comparisons to other gems.
```
Comment (justin808) [inline README.md]: This sort of md link won't work.

**Review by justin808 — COMMENTED** [review]
https://chrome.google.com/webstore/detail/link-checker/aibjbgmpmnidnmagaefhmcjhadpffaoi to find the bad links.

**Review by justin808 — COMMENTED** [review]
OK on this commit

**Review by justin808 — COMMENTED** [review]
@Judahmeek did you check that any recent updates to the README and other docs got merged correctly into this PR?

File: README.md (hunk 1 above)
Comment (Judahmeek) [reply README.md]: ...and that's why relative links won't work right now.

**Comment by Judahmeek** [comment]
> @Judahmeek did you check that any recent updates to the README and other docs got merged correctly into this PR?

I reverted [your Status Hero image update](https://github.com/shakacode/react_on_rails/commit/c8d3837cfc8976a32d6c873a655adeb71ac989a2#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) during the rebase & forgot to check whether that was the correct choice so I'm glad you asked.

My latest commit corrects this & the shakacode `.md` link you mentioned above.

**Pull request merged**